server/services/applicationService.ts

```diff
@@ -104,20 +104,69 @@ export default class ApplicationService {
   }
 
   private deleteOrphanedFollowOnAnswers(applicationData: AnyValue): AnyValue {
-    if (applicationData['funding-information']?.['funding-source']?.fundingSource === 'personalSavings') {
+    const deleteOrphanedFundingInformation = () => {
       delete applicationData['funding-information'].identification
       delete applicationData['funding-information']['alternative-identification']
     }
-    if (applicationData['equality-and-diversity-monitoring']?.['will-answer-equality-questions']?.willAnswer === 'no') {
+
+    const deleteOrphanedEqualityInformation = () => {
       Object.keys(applicationData['equality-and-diversity-monitoring']).forEach(key => {
         if (key !== 'will-answer-equality-questions') {
           delete applicationData['equality-and-diversity-monitoring'][key]
         }
       })
     }
-    if (applicationData['offending-history']?.['any-previous-convictions']?.hasAnyPreviousConvictions === 'no') {
+
+    const deleteOrphanedOffendingHistoryInformation = () => {
```

Comment:
maybe not worth it (yet) but I this one and `deleteOrphanedFundingInformation` could probably be abstracted into one generic function that takes in the task name and the page names to delete data for?

Reply:
agreed, I think if we add one more in we should move to a utility